.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,24 +7,24 @@ assignees: ''
 
 ---
 
-**Describe the bug**
```

Comment:
Can you split RDS exporter metrics fix and Github PR template and rephase commits so both changes will appear in the change log?

You can found the commit syntax on https://github.com/qonto/prometheus-rds-exporter/blob/main/.goreleaser.yaml#L86-L106 and documentation https://goreleaser.com/customization/changelog/?h=changelog

Suggestion:

fix(metrics): remove missing dbidentifier from rds_instance_max_iops_average, rds_instance_max_throughput_bytes and  rds_instance_vcpu_average
fix(github): Define new template for Github issue

Reply:
Splitted with https://github.com/qonto/prometheus-rds-exporter/pull/126